frontend/src/views/LoginView.vue

```diff
@@ -1,22 +1,33 @@
 <template>
 	<div class="page_box_wrapper">
-        <h1> HI WANNA PLAY PONG? </h1>
-
-            <div class="c_block c_form_group">
-                <InputField label="Username" placeholder="username" v-model="loginForm.name"/>
-                <InputField label="Password" placeholder="password" v-model="loginForm.password"/>
-
-                <div class="c_block c_split">
-                    <p>
-                        <a href="#" v-on:click.prevent="userStore.login(loginForm)">Login</a>
-                        /
-                        <a href="/register">Register</a>
-                    </p>
-                    <p>
-                        <a :href="redirectUrl">Sign in with 42</a>
-                    </p>
-                </div>
-            </div>
+		<div class="page_box">
+          <!-- Content should be here -->
+		  <h1> HI WANNA PLAY PONG? </h1>
+		  
+		  <div class="c_block c_form_group">
+			  <form 
+			  method="post"
+					action=""
+					class="c_block c_form_group"
+					@submit.prevent="userStore.login(loginForm)"
+					>
+					
+					<InputField label="Username" placeholder="username" v-model="loginForm.name"/>
+					<InputField label="Password" placeholder="password" v-model="loginForm.password"/>
+					
+					<div class="c_block c_split">
+						<p>
+							<input v-on:submit="userStore.login(loginForm)" class="link_button" type="submit" value="Login" />
```

Comment:
is this second `v-on:submit=""` needed here? I thought it works without this (since the button submits the form, and the form already has a submit action)

Reply:
I checked, the v-on on the input is not needed !